src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/PopupOpenedEventArgs.shared.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	/// <summary>
+	/// Popup opened event arguments used when a popup is opened.
+	/// </summary>
+	public class PopupOpenedEventArgs : EventArgs
```

Comment:
The intent here was to create an event argument that we could build off in the future. Considering that all event arguments should inherit from `EventArgs` I agree that we should just remove this.

Reply:
If we will use this, will be better to keep it there, because adding in the future the users will need to go for all delegates and change the `EventArgs` to `PopupOpenedEventArgs `.